### ISSUE TITLE
Wavenet training loss no change around 7

### ISSUE THREAD

**Issue by zliuar** [Rayhane-mamah/Tacotron-2#96]
Hi, I am training the wavenet right now and I found the training loss is oscillating around 6.7 to 7 from 5.5k steps to 8.5k steps (now). The batch size is 4 as default. Is it normal? Anyone can show your wavenet triaining loss?

The parameters are all set as default. The wavenet is trained using /tacotron_output/gta/map.txt.

**Comment by zliuar** [comment]
Thanks! Will try smaller learning rate!
